### DISCUSSION TITLE
OAuth 第2步 获取授权码失败

### DISCUSSION THREAD

**Discussion by snowords** [spencerwooo/onedrive-vercel-index#581] [Q&A] [answered]
## 教育版账户，登录后跳转localhost地址，无法获取授权码
> 报错地址：
```
http://localhost/?error=invalid_client&error_description=AADSTS650051%3a+Using+application+%27my-onedrive-vercel-index%27+is+currently+not+supported+for+your+organization+mails.qust.edu.cn+because+it+is+in+an+unmanaged+state.+An+administrator+needs+to+claim+ownership+of+the+company+by+DNS+validation+of+mails.qust.edu.cn+before+the+application+my-onedrive-vercel-index+can+be+provisioned.%0d%0aTrace+ID%3a+3c63a65d-18c8-4046-b71d-e3b064316000%0d%0aCorrelation+ID%3a+df42ca0b-835d-4868-8610-b3eb89abbb3c%0d%0aTimestamp%3a+2022-04-02+03%3a35%3a10Z#
```
> 提取出来的错误信息为

Using application ”my-onedrive-vercel-index“ is currently not supported for your organization mails.qust.edu.cn because it is in an unmanaged state.An administrator needs to claim ownership of the company by DNS validation of mails.qust.edu.cn before the application my-onedrive-vercel-index can be provisioned.

---
设置 clientid 和 clientsecret的方法也试过了

**Comment by spencerwooo** [comment, marked as answer]
> An administrator needs to claim ownership of the company by DNS validation of mails.qust.edu.cn

这看样子需要校方给教育版账户开权限，否则就不能用了。建议用 E5 开发者版如果没有订阅 Office 365 的话。
